sklearn/linear_model/coordinate_descent.py

```diff
@@ -786,19 +786,19 @@ class Lasso(ElasticNet):
 
     Parameters
     ----------
-    alpha : float, optional
-        Constant that multiplies the L1 term. Defaults to 1.0.
+    alpha : float, default 1.0.
```

Comment:
I tried to be consistent with the other default parameters in the file; in the
```
parameter_name : type, default x
```
format. I usually try to do
```
parameter_name : type, (default=x)
```
in my other PRs. In any case, I do agree with you, and I'm surprised that scikit-learn has not established default parameter formats.

I'd like to keep this PR going. Is it ready to be merged, or how would you like me to go from here?